packages/conf-calcium/conf-calcium.1/opam

```diff
@@ -30,6 +30,12 @@ depexts: [
   ["calcium" "calcium-devel"] {os-distribution = "ol"}
   ["calcium-devel"] {os-family = "suse"}
   ["calcium"] {os-distribution = "nixos"}
+  ["calcium"] {os-distribution = "arch" }
```

Comment:
not sure this helps if `calcium` cannot be install by pacman, but it shouldn't hurt.

Reply:
If/when opam can configure the package manager, then that will work. In the meantime, I think users can already alias `pacman` to their pacman wrapper of choice to make that work (but I'll have to check once this PR is merged).